cibyl/sources/server.py

```diff
@@ -0,0 +1,42 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+
+from cibyl.exceptions.source import MissingArgument
+from cibyl.sources.source import Source
+
+LOG = logging.getLogger(__name__)
+
+
+class ServerSource(Source):
+    """A class representation of a source that must connect to a server
+    instance."""
+
+    # pylint: disable=too-many-arguments
+    def __init__(self, enabled: bool = True, priority: int = 0,
```

Comment:
I believe that if you do not define a constructor, you inherit the one from the parent class.

Reply:
That's a good point, since the constructor does not provide anything I've just removed it and use the parent one.